### PR TITLE
[optimize]Add Doris restart script, optimize doris start and stop commands

### PR DESCRIPTION
**Pull request by hf200012**
Add Doris restart script, optimize doris start and stop commands

Can be used directly:

```
sh bin/doris_fe.sh start | stop | restart
sh bin/doris_be.sh start | stop | restart
```
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [ ] Compiling and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If these changes need document changes, I have updated the document
- [ ] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: bin/doris_be.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```
Comment (morningman) [inline bin/doris_be.sh:1]: License header.

File: bin/doris_be.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+export DORIS_HOME=`cd "$curdir"; pwd`
+export LOG_DIR=${DORIS_HOME}/log
```
Comment (morningman) [inline bin/doris_be.sh:4]: the `LOG_DIR` should read from config file.

File: bin/doris_be.sh [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    else
+       echo "Apache Doris Backend stop success!"
+    fi
+}
+
+function restart() {
+    echo "Restart Apache Doris Backend"
+    echo "****************************"
+    echo "Ready to stop Doris Backend"
+    stop
```
Comment (morningman) [inline bin/doris_be.sh:34]: The process may be in defunct status for a long time, so start right after stop may failed to start BE process.

File: bin/doris_be.sh [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+function start() {
+    echo "Starting Apache Doris Frontend"
+    sh $DORIS_HOME/bin/start_be.sh --daemon
+    if [ -f "$DORIS_HOME/bin/be.pid" ]; then
+       echo "Apache Doris Backend start fail!"
+    else
+      sleep 1s
+      pid=`cat $DORIS_HOME/bin/be.pid`
+      echo "Doris Be pid : ${pid}"
+      echo `sed -n '$p' $DORIS_HOME/log/be.out`
```
Comment (morningman) [inline bin/doris_be.sh:15]: the log dir may be elsewhere

File: bin/doris_be.sh [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+export DORIS_HOME=`cd "$curdir"; pwd`
+export PID_DIR=`cd "$curdir"; pwd`
+function start() {
+    echo "Starting Apache Doris Frontend"
```
Comment (morningman) [inline bin/doris_be.sh:22]:
```suggestion
    echo "Starting Apache Doris Backend"
```

File: build.sh [hunk 6]
```diff
@@ -327,6 +327,7 @@ if [ ${BUILD_BE} -eq 1 ]; then
                ${DORIS_OUTPUT}/udf/include
 
     cp -r -p ${DORIS_HOME}/be/output/bin/* ${DORIS_OUTPUT}/be/bin/
+    cp -r -p ${DORIS_HOME}/bin/doris_be.sh ${DORIS_OUTPUT}/be/bin/
```
[the remark this answers is not in the document]
Comment (hf200012) [reply build.sh:330]: Missed one

[the remark this answers is not in the document]
Comment (hf200012) [reply build.sh:330]: > 
cp -r -p ${DORIS_HOME}/bin/*_fe.sh ${DORIS_OUTPUT}/fe/bin/

**Pull request closed without merging**